### ISSUE TITLE
No such module 'Kickstarter_iOS_Sources' upon running "_Introduction playground"

### ISSUE THREAD

**Issue by daurenm** [kickstarter/ios-oss#390]
I couldn't run any of playgrounds, even though the app itself compiles and runs on a device. The following is the error I keep getting:
`
Playground execution failed:
error: /var/folders/4l/frjs5bv97kl4dkysttm2z_440000gn/T/playground63-29e196..swift:3:8: error: no such module 'Kickstarter_iOS_Sources'
import Kickstarter_iOS_Sources
`
Here is the content of that file:
`
@_silgen_name ("playground_logger_initialize") func $builtin_logger_initialize ()
@_silgen_name ("playground_log_hidden") func $builtin_log_with_id<T> (_ object : T, _ name : String, _ id : Int, _ sl : Int, _ el : Int, _ sc : Int, _ ec: Int) -> AnyObject
@_silgen_name ("playground_log_scope_entry") func $builtin_log_scope_entry (_ sl : Int, _ el : Int, _ sc : Int, _ ec: Int) -> AnyObject
@_silgen_name ("playground_log_scope_exit") func $builtin_log_scope_exit (_ sl : Int, _ el : Int, _ sc : Int, _ ec: Int) -> AnyObject
@_silgen_name ("playground_log_postprint") func $builtin_postPrint (_ sl : Int, _ el : Int, _ sc : Int, _ ec: Int) -> AnyObject
@_silgen_name ("DVTSendPlaygroundLogData") func $builtin_send_data (_ :  AnyObject!)
$builtin_logger_initialize()
import Kickstarter_iOS_Sources
#sourceLocation(file: "Kickstarter-iOS.playground", line: 1)
print("Hello!")
`

I guess, I'm not compiling proper target to run them (I compiled Kickstarter-Framework-iOS). Does someone have any ideas on what I'm doing wrong?)

**Comment by amadeu01** [comment]
It seems that they do not really have put much effort into support playgrounds anymore... 😢

**Comment by justinswart** [comment]
Playgrounds in Xcode are extremely buggy and live views in particular can be unreliable. I had a look into this issue and could get one or two of our playgrounds to run based on steps in two Stackoverflow answers:

1) Quit _all_ versions of Xcode currently running and close all open projects.
2) Open _only_ the Kickstarter project.
3) Navigate to the Playground of interest (eg. `Settings`)
4) Open the Utilities panel (opt+cmd+0)
5) Under `Playground Settings` toggle the Platform dropdown from iOS -> macOS and then back to iOS.
6) Try to run the Playground.

I have varying success doing these steps, also once I got the Playground to run I had to repeat all 6 steps again to run the next one.

Relevant SO answers:

https://stackoverflow.com/a/43022086/2271759
https://stackoverflow.com/a/47311437/2271759

Closing this for now as this is really an Xcode problem and not specific to our project.
